packages/react/src/components/progress/progress.stories.tsx

```diff
@@ -3,6 +3,8 @@ import React, { useEffect, useState } from 'react';
 import { Meta, omit, reactMatrix, Story } from '../../../../../docs';
 
 const size = ['regular', 'large'] as const;
+size.toString = () => size.map((value) => `"${value}"`).join('|');
```

Comment:
This stringifier function could be a helper, since we need it in several stories.

Reply:
Did create a helper for this, please confirm (last commit) if it's OK, or you'd suggest something different.

Reply:
Looks good, left a comment there.